meta/main.yml

```diff
@@ -15,10 +15,10 @@ galaxy_info:
   platforms:
   - name: EL
     versions:
-    - all
```

Comment:
Platforms updated: centos 6 and 7, amazon all. People will need to trust us about amazon, because we can't get travis to run it.